clients/vscode-hlasmplugin/schema/proc_grps.schema.json

```diff
@@ -36,6 +36,11 @@
                                             "items": {
                                                 "type": "string"
                                             }
+                                        },
+                                        "prefer_alternate_root": {
+                                            "type": "boolean",
+                                            "description": "Indicates that relative paths should be evaluated with respect to an alternative root folder (e.g. provided by the Endevor Bridge for Git).",
```

Comment:
Specify "true" to resolve relative paths with respect to a different root folder than the workspace root (e.g. a folder provided by Endevor Bridge for Git)